docs/modules/ROOT/pages/how-tos/_nav.adoc

```diff
@@ -12,6 +12,7 @@
 *** xref:how-tos/configuring/build-with-args.adoc[Passing buildah arguments]
 *** xref:how-tos/configuring/custom-tags.adoc[Using custom tags]
 *** xref:how-tos/configuring/dynamic-labels.adoc[Using dynamic labels]
```

Comment:
After thinking about this a little more, how would you feel about merging this new content with the labels page? That page isn't _entirely_ about dynamic labels. We could change the name of it to just refer to build metadata, defining labels and annotations, or something similar. After that, we would have a section on labels, annotations, and then dynamic labels (with further subsections).

The dynamic labels are just just key-value pairs, so we might even be able to take advantage of the dynamic labels flow to add dynamic annotations.

Reply:
That's a great idea, it should work with both

Reply:
Better now?